pkg/commands/root.go

```diff
@@ -84,6 +84,7 @@ func (e *Executor) needVersionOption() bool {
 
 func initRootFlagSet(fs *pflag.FlagSet, cfg *config.Config, needVersionOption bool) {
 	fs.BoolVarP(&cfg.Run.IsVerbose, "verbose", "v", false, wh("verbose output"))
+	fs.BoolVarP(&cfg.Run.DisableCongrats, "no-congrats", "s", false, wh("disable congrats output"))
```

Comment:
1. maybe `silent` instead of `no-congrats` and `Silent` instead of `DisableCongrats`? It will be more general logic. Also in some places options disables not only congrats message, e.g. in JSON
2. `s/disable/Disable`

Reply:
Sure it makes sense.